tasks/tasks_test.go

```diff
@@ -199,15 +357,15 @@ func (db *FakeDatabase) StageFiles(fileIds []string) (uuid.UUID, error) {
 	return id, nil
 }
 
-func (db *FakeDatabase) StagingStatus(id uuid.UUID) (StagingStatus, error) {
+func (db *FakeDatabase) StagingStatus(id uuid.UUID) (core.StagingStatus, error) {
 	if info, found := db.Staging[id]; found {
 		if time.Now().Sub(info.Time) >= stagingDuration {
-			return StagingStatusSucceeded, nil
+			return core.StagingStatusSucceeded, nil
 		} else {
-			return StagingStatusActive, nil
+			return core.StagingStatusActive, nil
```

Comment:
no change required, but just to note that there's no need for the `else` if you `return`ed in the `if` code.